app/presenters/speedy_af/proxy/media_object.rb

```diff
@@ -140,6 +140,22 @@ def governing_policies
     @governing_policies ||= Array(attrs[:isGovernedBy]).collect { |id| SpeedyAF::Base.find(id) }
   end
 
+  def language
+    attrs[:language_code].map { |code| { code: code, text: LanguageTerm.find(code).text } } if attrs[:language_code].present?
+  end
+
+  def note
+    attrs[:note].map.with_index { |n, i| { note: n, type: attrs[:note_type][i] } } if attrs[:note].present?
+  end
+
+  def other_identifier
+    attrs[:other_identifier].map.with_index { |oi, i| { id: oi, source: attrs[:other_identifier_type][i] } } if attrs[:other_identifier].present?
+  end
+
+  def related_item_url
+    attrs[:related_item_url].map.with_index { |url, i| { url: url, label: attrs[:related_item_label][i] } } if attrs[:related_item_url].present?
+  end
```

Comment:
As mentioned in standup, these are the fields that I am concerned about content drift on. In automated testing, and in the limited manual testing I have done, the order of the SOLR fields is consistent and these methods return the proper `{ key: value }` pairs. Is it possible that the fields can end up out of sync? What other approaches might we be able to take to ensure these fields return the same information as calls to the Media Object itself?